src/api/GlobalTracer.php

```diff
@@ -45,3 +45,6 @@ public static function get()
         return self::$instance = NoopTracer::create();
     }
 }
+
+// Ensure that, when trying to use the legacy API, our Tracer is also loaded
+class_exists('DDTrace\\Tracer');
```

Comment:
It still does. When the extension is not installed, Tracer does not exist and class_exists will do exactly nothing. it's "try invoking autoloader but don't do anything if not there".
This is crucial to the PR, so that, _if_ the extension is enabled, the user can to `GlobalTracer::get()` and get a proper Tracer.

Reply:
> Tracer does not exist and class_exists will do exactly nothing

Unfortunately in the past we had issues with customers having autoloaders that 'throw' if they do not find a class. Even recently we had a similar issue with https://github.com/DataDog/dd-trace-php/pull/1451 (in this case was magento). So this mechanism cannot be used. It is pretty late and I might be wrong 😄  but I think we would get the very same error

Reply:
```suggestion
try {
    class_exists('DDTrace\\Tracer');
} catch (\Exception $e) {
} catch (\Throwable $e) {
}
```
Would that work out? just swallowing throwing autoloaders?

Reply:
I don't think so. Some autoloaders might not "throw" in the sense that throw an exception. For example Magento's autoloader directly loads the file with `include`, so we would cause a warning to be emitted (for example):
```
Warning: include(/home/circleci/app/playground/file_that_doesnt_exist.php): failed to open stream: No such file or directory in /home/circleci/app/playground/index.php on line 13

Warning: include(): Failed opening '/home/circleci/app/playground/file_that_doesnt_exist.php' for inclusion (include_path='.:/opt/php/debug/lib/php') in /home/circleci/app/playground/index.php on line 13
```
Let me time cap playing with this to 15 mins if I can propose something, rather than only saying 'that does not work' 😄

Reply:
As we agreed in chat and for future reviewer's memory:
1. we define a function `function trigger_legacy_tracer_autoload() {}` in `bridge/autoload.php`;
2. we change `GlobalTracer::get()` to be
```
    public static function get()
    {
        if (null !== self::$instance) {
            return self::$instance;
        }

        if (\extension_loaded('ddtrace') && function_exists('trigger_legacy_tracer_autoload')) {
            \DDTrace\Bootstrap::tracerOnce();
            if (null !== self::$instance) {
                return self::$instance;
            }
        }

        return self::$instance = NoopTracer::create();
    }
```
3. we remove `BootstrapTracer::once()` from `Tracer.php` file and `class_exists(...)` from `GlobalTracer`.